xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -69,7 +101,7 @@
         <ReturnType>System.Runtime.Loader.AssemblyLoadContext</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the instance of the default <see cref="T:System.Runtime.Loader.AssemblyLoadContext" />.  The default context contains the framework and the main application assembly and its dependencies.</summary>
```

Comment:
Nit: "Instance of the default ALC" is a bit confusing, since default ALC itself is an instance.